pkg/scheduler/metrics/metrics.go

```diff
@@ -44,6 +42,22 @@ var (
 			Buckets:   prometheus.ExponentialBuckets(1000, 2, 15),
 		},
 	)
+	SchedulingAlgorithmPredicateEvaluationDuration = prometheus.NewHistogram(
+		prometheus.HistogramOpts{
+			Subsystem: schedulerSubsystem,
+			Name:      "scheduling_algorithm_predicate_evaluation",
+			Help:      "Scheduling algorithm predicate evaluation duration",
+			Buckets:   prometheus.ExponentialBuckets(1000, 2, 15),
```

Comment:
It would be good to add a comment here and below to mention that our smallest bucket is 1 mili-second.
Otherwise, LGTM.

Reply:
Done @bsalamat - I have added a top level comment for all the histogram based buckets.